py-polars/polars/series/series.py

```diff
@@ -1421,14 +1421,16 @@ def quantile(
         """
         return self._s.quantile(quantile, interpolation)
 
-    def to_dummies(self, separator: str = "_") -> DataFrame:
+    def to_dummies(self, separator: str = "_", include_null: bool = False) -> DataFrame:
```

Comment:
Boolean flags should  be keyword-only:

```suggestion
    def to_dummies(self, separator: str = "_", *, include_null: bool = False) -> DataFrame:
```